src/components/components/Component.js

```diff
@@ -176,17 +232,27 @@ export default class Component extends React.Component {
         if (whichAssetType == "background"){
           whichOptions = this.state.backgroundList;
         }
+        const objId = this.props.entity.getAttribute("id");//.replace("-obj", "");
         return (
-          <Select
-            styles={customStyles}
-            value={this.state.nameList.filter(option => option.value == componentData.data)}
-            ref="select"
-            options={whichOptions}
-            placeholder="Add component..."
-            noResultsText="No components found"
-            searchable={true}
-            onChange={this.selectOption}
-          />
+          <div>
+            <Select
+              styles={customStyles}
+              value={this.state.nameList.filter(option => option.value == componentData.data)}
+              ref="select"
+              options={whichOptions}
+              placeholder="Add component..."
+              noResultsText="No components found"
+              searchable={true}
+              onChange={this.selectOption}
+            />
+            <GltfPopUp
```

Comment:
We should also wrap this in a conditional, so it only creates the GltfPopup if `objId.endsWith("-obj")`